core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala

```diff
@@ -87,6 +113,8 @@ class StorageStatusListener extends SparkListener {
   override def onBlockManagerRemoved(blockManagerRemoved: SparkListenerBlockManagerRemoved) {
     synchronized {
       val executorId = blockManagerRemoved.blockManagerId.executorId
+      removedExecutorIdToStorageStatus.put(executorId,
+          executorIdToStorageStatus.get(executorId).get)
```

Comment:
Two things:
1. For that, I'll have to add a new column in the `execTable` on the UI. Should it be lastStatusChangedTime or KilledTime with blank values for the Alive executors?
2. This value would always be greater than the value `currentTime - spark.ui.timeToExpireKilledExecutor`. I am not really sure, we provide any useful insights by showing the time at which executor died.